### PR TITLE
Readme add

### PR DESCRIPTION
**Pull request by Shaquon**
Added a readme with a title and description.

### CODE REVIEW

**Review by shaquonkelley — COMMENTED** [review]
Hey @Shaquon I like this change. I would also work on adding a link to the the git command cheat sheet.

**Comment by Shaquon** [comment]
ok @shaquonkelley I will make that change now. Thanks you.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# git-revisited
+
+# making changes
```
Comment (Shaquon) [inline README.md:3]: I'm working on this now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# git-revisited
```
Comment (Shaquon) [inline README.md:1]: this will be another change

File: README.md (hunk 1 above)
Comment (shaquonkelley) [reply README.md:3]: ok that's good.

**Review by Shaquon — COMMENTED** [review]
final change made

File: README.md (hunk 2 above)
Comment (shaquonkelley) [reply README.md:1]: ok

**Pull request merged**